src/js/formats/bigNumberCarousel.js

```diff
@@ -1,129 +1,38 @@
-import bigNumberCarouselTemplate from '../text/bigNumberCarousel.dot.html!text';
-import q from '../lib/query';
+import bigNumberCarouselTemplate from '../text/carousel.dot.html!text';
```

Comment:
Should the import name change to reflect that it's not necessarily a big number any more?